.evergreen.yaml

```diff
@@ -115,32 +133,8 @@ functions:
         script: |
           export PATH="${workdir}/bin:$PATH"
 
-          echo "Install kubectl"
-          curl -L -o bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
-          chmod +x bin/kubectl
-
-          echo "Install kind"
-          curl -L -o bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-linux-amd64
-          chmod +x bin/kind
-
-          echo "Start cluster"
-          kind create cluster 
-          export KUBECONFIG="$(kind get kubeconfig-path)"
-
-          echo "Install Helm"
-          curl -LO https://get.helm.sh/helm-v2.14.3-linux-amd64.tar.gz
-          tar -zxvf helm-v2.14.3-linux-amd64.tar.gz
-          mv linux-amd64/helm bin/helm
-          
-          echo "Setup helm"
-          kubectl --namespace kube-system create serviceaccount tiller
-          kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
-          helm init --service-account tiller --wait
-
-          echo "Install Service Catalog extension"
-          helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
-          helm install svc-cat/catalog --name service-catalog --namespace catalog
-          kubectl rollout status --watch deployment/service-catalog-catalog-apiserver --namespace=catalog
+          # Setup local Kubernetes cluster
+          ./src/atlas-service-broker/dev/scripts/evergreen-kubernetes-setup.sh
```

Comment:
I'd probably prefer if all of these commands were in the dedicated script. My general philosophy would be that if there is more than one statement that has a possibility of failing then they should be in a script file that can be independently executed and debugged.

Reply:
+1

Reply:
You mean the "Adding docker image to cluster" commands? I left them out because they make use of expansions but I guess I could make the script accept args.

Reply:
Yeah. I'd say either have it accept arguments or explicitly pass the environment variables into the script, i.e.
```
export MY_EXPANSION=${MY_EXPANSION}
./run-my-script
```

Reply:
Got it!